checksum.go

```diff
@@ -183,15 +184,15 @@ func newChecksumFromValue(checksumValue, filename string) (*FileChecksum, error)
 	return c, nil
 }
 
-// ChecksumFromFile will return all the FileChecksums found in file
+// ChecksumFromFile will return the first FileChecksums found in file.
 //
-// ChecksumFromFile will try to guess the hashing algorithm based on content
-// of checksum file
+// ChecksumFromFile will try to guess the hashing algorithm based on the
+// content of a checksum file.
```

Comment:
My English is failing me here but a tad vague, it's not immediately obvious which `file` this comment we refer to - maybe the following:

```
// ChecksumFromFile will return the first file checksum found in `checksumURL` file
// that correspond to the `checksummedURL` path.
//
// ChecksumFromFile will infer the hashing algorithm based on the checksumURL file content.
```

Reply:
Oh super good, changing this too.